templates/modals/delete_doc_confirm.html

```diff
@@ -1,15 +1,12 @@
+<!-- TODO fix title and submit translations -->
```

Comment:
What's meant by "fix title"? And does it need to happen before merging?

Reply:
Sorry, that was a TODO I missed before merging. It's done, the titles are correct, I just forgot to remove the reminder